src/server/pkg/hashtree/cache.go

```diff
@@ -4,35 +4,124 @@ import (
 	"fmt"
 	"io"
 	"reflect"
+	"sync"
 
-	lru "github.com/hashicorp/golang-lru"
+	"github.com/hashicorp/golang-lru/simplelru"
 	"github.com/pachyderm/pachyderm/src/server/pkg/localcache"
 	"github.com/sirupsen/logrus"
 )
 
+type nascentTree struct {
```

Comment:
Should we call a nascent tree a `seedling`? I believe that's the technical term :stuck_out_tongue:

Reply:
I'm not sure if this what just a joke, but I liked it and did it anyway.